src/components/WebexLocalMedia/WebexLocalMedia.scss

```diff
@@ -10,6 +10,12 @@
 
   background: black;
 
+  .my-preview {
+    position: absolute;
+    top: 25%;
```

Comment:
isn't the banner supposed to be displayed at the top of the local media component?